//public class Lab9Task2_Exception {
//    public static void main(String[] args) {
//        try {
//
//        } catch (Exception e) {
//            {
//
//            }
//
//        }
//
//
//}
